plugins/modules/cpanm.py

```diff
@@ -56,6 +56,12 @@
       - Only install dependencies.
     type: bool
     default: false
+  withrecommands:
+    description:
+      - Installs dependencies declared as recommends per META spec.
+      - When these dependencies fail to install, cpanm continues the installation, since they're just recommendation/suggestion.
```

Comment:
```suggestion
      - When these dependencies fail to install, cpanm continues the installation, since they are just recommendation/suggestion.
```

Reply:
In fact I would even rather do:

```suggestion
      - When these dependencies fail to install, cpanm continues the installation, since they are just recommendation.
```

Only recommandation, not suggestion. That would be for a future integration of the option, i.e ``--with-suggests``